src/main/java/com/google/sps/placeGuide/repository/impl/DatastorePlaceGuideRepository.java

```diff
@@ -84,6 +86,58 @@ private Entity createPlaceGuideEntity(PlaceGuide placeGuide) {
     return getPlaceGuidesList(query);
   }
 
+   @Override
+  public void deletePlaceGuide(long placeGuideId) {
+    Key placeGuideEntityKey = KeyFactory.createKey(ENTITY_KIND, placeGuideId);
+    datastore.delete(placeGuideEntityKey);
+  }
+
+  public List<PlaceGuide> getAllPublicPlaceGuidesInMapArea(GeoPt northEastCorner,
```

Comment:
@Override

Reply:
Just because I forgot about it. Thanks for reminding me, I added it now. :)